doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,302 @@
+
+* Added the following libraries:
+
+  * :ref:`bas_c_readme` - used to retrieve information about the battery level from a device.
+  * :ref:`shell_bt_nus_readme` - allows for sending shell commands from a host to the application.
+
+* Added :ref:`ble_console_readme` - a desktop application that can be used to communicate with an nRF device over *Bluetooth* Low Energy using the :ref:`shell_bt_nus_readme`.
+
+NFC
+---
+
+*
+
+Partition Manager
+-------------
+
+* Partition Manager can now implicitly or explicitly assign a HEX file to a partition.
+* :ref:`ug_pm_static` of upgradable images is now supported.
+
+
+nRF Desktop
+===========
+
+* The nrf_desktop reference implementation is moved from the ``samples/`` folder to ``applications/``.
+* The nrf_desktop configuration channel now allows data to be exchanged between the device and host in both directions.
+
+Build and configuration system
+==============================
+
+???
+
+Documentation  - WORK IN PROGRESS
+=============
+
+* Added or updated documentation for the following samples:
+
+  *
+  *
+  *
+  *
+
+* Added or updated documentation for the following libraries:
+
+  *
+  *
+  *
+  *
+  *
+  *
+  *
+  *
+  *
+
+* Added or updated the following documentation:
+
+  *
+  *
+  *
+  *
+  *
+  *
+
+
+Known issues
+************
+
+nRF9160
+=======
+
+* The :ref:`asset_tracker` sample does not wait for connection to nRF Cloud before trying to send data.
+  This causes the sample to crash if the user toggles one of the switches before the board is connected to the cloud.
+* The :ref:`asset_tracker` sample might show up to 2.5 mA current consumption in idle mode with ``CONFIG_POWER_OPTIMIZATION_ENABLE=y``.
+* If a debugger (for example, J-Link) is connected via SWD to the nRF9160, the modem firmware will reset.
+  Therefore, the LTE modem cannot be operational during debug sessions.
+* The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+  As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+  If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+
+Crypto
+======
+
+* The :ref:`nrfxlib:nrf_security` glue layer is broken because symbol renaming is not handled correctly.
+  Therefore, the behavior is undefined when selecting multiple back-ends for the same algorithm (for example, AES).
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* :ref:`peripheral_lbs` does not report the Button 1 state correctly.
+* The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+* The :ref:`gatt_dm_readme` is not supported on nRF51 devices.
+* Bluetooth LE samples cannot be built with the :ref:`nrfxlib:ble_controller` v0.1.0.
```

Comment:
I think that this line should be removed because we migrated to the v0.2.0 of the controller

Reply:
removed